src/test/java/com/commercetools/sync/it/products/SphereClientUtils.java

```diff
@@ -0,0 +1,53 @@
+package com.commercetools.sync.it.products;
+
+import com.commercetools.sync.commons.utils.ClientConfigurationUtils;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereClientConfig;
+import io.sphere.sdk.client.SphereRequest;
+import io.sphere.sdk.queries.PagedQueryResult;
+
+import javax.annotation.Nonnull;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+public class SphereClientUtils {
```

Comment:
Could you put a respective comment, like: _TODO: re-factor with inventory/categoriest test sphere utils, when merged_

Thanks